sample-code/src/orchestration.ts

```diff
@@ -1,133 +1,171 @@
 import {
+  LlmModuleConfig,
   OrchestrationClient,
+  OrchestrationResponse,
   buildAzureContentFilter
 } from '@sap-ai-sdk/orchestration';
+import { createLogger } from '@sap-cloud-sdk/util';
 
-/**
- * Create different types of orchestration requests.
- * @param sampleCase - Name of the sample case to orchestrate.
- * @returns The message content from the orchestration service in the generative AI hub.
- */
-export async function orchestrationCompletion(
-  sampleCase: string
-): Promise<string | undefined> {
-  switch (sampleCase) {
-    case 'simple':
-      return orchestrationCompletionSimple();
-    case 'template':
-      return orchestrationCompletionTemplate();
-    case 'filtering':
-      return orchestrationCompletionFiltering();
-    case 'requestConfig':
-      return orchestrationCompletionRequestConfig();
-    default:
-      return undefined;
-  }
-}
+const logger = createLogger({
+  package: 'sample-code',
+  messageContext: 'orchestration'
+});
 
 /**
- * Ask about the capital of France.
- * @returns The message content from the orchestration service in the generative AI hub.
+ * A simple LLM request, asking about the capital of France.
+ * @returns The orchestration service response.
  */
-async function orchestrationCompletionSimple(): Promise<string | undefined> {
+export async function orchestrationChatCompletion(): Promise<OrchestrationResponse> {
   const orchestrationClient = new OrchestrationClient({
+    // define the language model to be used
     llm: {
-      model_name: 'gpt-4-32k',
+      model_name: 'gpt-4o',
       model_params: {}
     },
+    // define the prompt
     templating: {
       template: [{ role: 'user', content: 'What is the capital of France?' }]
     }
   });
 
-  // Call the orchestration service.
-  const response = await orchestrationClient.chatCompletion();
-  // Access the response content.
-  return response.getContent();
+  // execute the request
+  const result = await orchestrationClient.chatCompletion();
```

Comment:
[pp] I would use `response` instead of `result` to keep it consistent with what we used in e.g. unit tests.

Reply:
For unit test that makes sense to me, for sample code however I persoanlly prefer `result` because response might be confused with an HTTP response.